services/brig/test/integration/API/User/Auth.hs

```diff
@@ -586,7 +586,7 @@ testRegularUserLegalHoldLogin brig = do
   uid <- userId <$> randomUser brig
   -- fail if user is not a team user
   legalHoldLogin brig (LegalHoldLogin uid (Just defPassword) Nothing) PersistentCookie !!! do
-    const 403 === statusCode
+    const 404 === statusCode
```

Comment:
@dkovacevic do you check the specifics of 4xx when logging in LH devices?  we're changing the status code for an error condition from 403 to 404 here, any objections?